docs/experience/breakdown.md

```diff
@@ -0,0 +1,30 @@
+title: Sematext Experience Detailed Timing Breakdown
+description: Investigate page loads and HTTP Requests performance breakdown with Sematext Experience
+
+Load time performance depends on multiple factors. If your visitors are experiencing slow page load or HTTP requests performance it can be caused by the network, DNS servers, redirects, application performance, etc. To better understand which part of the loading process is slow, we've added detailed timing breakdown on Page Loads and HTTP Requests screens.
```

Comment:
Load time performance depends on multiple factors.
=>
Load-time performance depends on multiple factors.

If your visitors are experiencing slow page load or HTTP requests performance it can be caused by the network, DNS servers, redirects, application performance, etc.
=>
If your visitors are experiencing slow page-loads or poor HTTP request performance, it's often caused by the network, DNS servers, redirects, application performance, among other things.

To better understand which part of the loading process is slow, we've added detailed timing breakdown on Page Loads and HTTP Requests screens.
=>
To better understand which part of the loading process is slow, we've added a detailed timing breakdown on the Page Loads and HTTP Requests screens.